translate/convert/test_prop2po.py

```diff
@@ -275,6 +277,18 @@ def test_successive_gaia_plurals(self):
         assert not zero_unit.hasplural()
         assert zero_unit.source == u"Edit"
 
+    @mark.xfail(reason="Still not fixed")
+    def test_duplicate_keys(self):
```

Comment:
Not sure if there are other tests, but I'd add all possible variants to document what we expect to happen:

```properties
key=value
key=another value
```

```properties
key1=value
key2=value
```